### DISCUSSION TITLE
Tutorial on aspect ratios in WordPress 6.6

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#281] [Topic Ideas]
This is a new feature landing in WordPress 6.6 for theme authors: https://make.wordpress.org/core/2024/06/24/miscellaneous-editor-changes-in-wordpress-6-6/#Add-custom-Aspect-Ratio-presets-through-theme-json

I propose a simple walkthrough tutorial of how to implement custom aspect ratios via `theme.json` or theme style variations. The tutorial could even include they can be used in patterns/templates. This feature allows users to use these aspect ratios with blocks that support them, such as Image, Featured Image, and Cover. _Note that wide/full Image blocks do not currently support setting an aspect ratio.

Example of custom ratios as shown in the editor:

![image](https://github.com/WordPress/developer-blog-content/assets/1816309/a4e67178-474d-4a68-8d1b-c5306bdf2657)

This should be pretty low-hanging fruit in terms of a tutorial.

**Comment by bph** [comment]
Topic was approved. Continue conversation on the issue
-#284

**Comment by colorful-tones** [comment]
Closing this out. It seems to have been published.
